### ISSUE TITLE
Support multiple PHP versions

### ISSUE THREAD

**Issue by philipnewcomer** [Pilothouse-App/pilothouse#23]
Support multiple versions of PHP, switchable via a cookie like Xdebug is currently toggled.

**Comment by AaronHolbrook** [comment]
Also a query arg would be nice for this

**Comment by AaronHolbrook** [comment]
Would also be nice to not install a ton of php version by default, but instead only if user directs dockerbox to do so (goes back to the idea of somehow allowing customization by user)

**Comment by philipnewcomer** [comment]
Added support for multiple versions of PHP, switchable via a `php` cookie and a query arg, similar to how Xdebug works already. The current Xdebug toggle/query arg plays nicely with the PHP toggle/query arg.

I decided to officially support in Pilothouse only the versions of PHP currently supported by the PHP maintainers themselves, 5.6, 7.0, and 7.1 currently. While I wouldn't rule out older versions of PHP, there are no official Docker images for unsupported PHP versions, so they will be much more difficult to support in Pilothouse.

**Documentation**
Changing the PHP version for web requests:
https://github.com/Pilothouse-App/pilothouse/wiki/Changing-a-site's-PHP-version
Changing the PHP version for WP-CLI commands:
https://github.com/Pilothouse-App/pilothouse/wiki/Using-WP-CLI
